src/main/resources/broad_secret.json

```diff
@@ -7,6 +7,8 @@
     "redirect_uris": [
       "urn:ietf:wg:oauth:2.0:oob",
       "http://localhost"
-    ]
+    ],
+    "client_id": "436986732241-nvjfkarirkhmcq7m3r1uemdau518uvdo.apps.googleusercontent.com",
```

Comment:
i assume you didn't mean to add these?

Reply:
+1 
please revert changes to broad_secret & verily_secret files
I have https://verily.atlassian.net/browse/BENCH-530 to address this

Reply:
Thanks! Reverted and updated my branch with the merge from main.